src/pages/docs/chat/rooms/history.mdx

```diff
@@ -3,7 +3,7 @@ title: Message storage and history
 meta_description: "Retrieve previously sent messages from history."
 ---
 
-The history feature enables users to retrieve messages that have been previously sent in a room. Ably stores chat messages for 30 days by default. You can extend this up to 365 days by [contacting us](https://forms.gle/SmCLNFoRrYmkbZSf8).
+The history feature enables users to retrieve messages that have been previously sent in a room. Ably stores chat messages for 30 days by default. You can extend this up to 365 days by [contacting us](https://ably.com/contact).
```

Comment:
Would you mind changing this to `/support`? See https://github.com/ably/docs/pull/2903

Reply:
Done! :)